source/fem-dem/vans_assemblers.cc

```diff
@@ -924,7 +962,7 @@ GLSVansAssemblerDallavalle<dim>::calculate_particle_fluid_interactions(
       // Particle's Reynolds number
       double re = 1e-1 + relative_velocity.norm() *
                            particle_properties[DEM::PropertiesIndex::dp] /
-                           physical_properties.fluids[0].viscosity;
+                           viscosity;
```

Comment:
```suggestion
                           (viscosity + DBL_MIN);
```

Reply:
The viscosity is always defined positive, why add a DBL_MIN? I don't think the model makes much sense with viscosity = 0 ?

Reply:
In the 3d simulation I'm doing to find the pressure difference between inside and outside of a bubble, the viscosity of the phases are equal to zero (so that the NS simulations can be solved analytically). But if you think it's unnecessary, we can ignore it.

Reply:
Ohh ok, I did not think about it this way. I'll add this then.